packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -59,22 +59,20 @@ abstract contract MajorityVotingBase is
     /// @notice The [ERC-165](https://eips.ethereum.org/EIPS/eip-165) interface ID of the contract.
     bytes4 internal constant MAJORITY_VOTING_INTERFACE_ID = type(IMajorityVoting).interfaceId;
 
-    /// @notice The ID of the permission required to call the `changeVoteSettings` function.
-    bytes32 public constant CHANGE_VOTE_SETTINGS_PERMISSION_ID =
-        keccak256("CHANGE_VOTE_SETTINGS_PERMISSION");
+    /// @notice The ID of the permission required to call the `updateMajorityVotingSettings` function.
+    bytes32 public constant UPDATE_MAJORITY_VOTING_SETTINGS_PERMISSION_ID =
```

Comment:
how about `UPDATE_VOTING_SETTINGS_PERMISSION`?

Reply:
Amended in c69aa38770f33a15baed838deca116c0c78f7e22